src/main/java/tahub/contacts/storage/JsonAdaptedGrade.java

```diff
@@ -0,0 +1,60 @@
+package tahub.contacts.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import tahub.contacts.commons.exceptions.IllegalValueException;
+
+/**
+ * Jackson-friendly version of a grade entry.
+ */
+class JsonAdaptedGrade {
+
+    private final String assessmentName;
+    private final double score;
```

Comment:
I think score is implemented as a percentage, might wanna consider renaming to something like ```scorePercentage``` to highlight that fact. (since not all scores are out of 100 marks).